examples/others/raylib_opengl_interop.c

```diff
@@ -26,21 +26,29 @@
 
 #include "raylib.h"
 
-#include "rlgl.h"           // Required for: rlDrawRenderBatchActive(), rlGetMatrixModelview(), rlGetMatrixProjection()
-#if defined(__APPLE__)
-    #include <OpenGL/gl3.h>     // OpenGL 3 library for OSX
-    #include <OpenGL/gl3ext.h>  // OpenGL 3 extensions library for OSX
-#else
-    #include "glad.h"       // Required for: OpenGL functionality 
-#endif
-#include "raymath.h"        // Required for: MatrixMultiply(), MatrixToFloat()
-
 #if defined(PLATFORM_DESKTOP)
-    #define GLSL_VERSION            330
+    #if defined(GRAPHICS_API_OPENGL_ES2)
+        #include "gles2.h"       // Required for: OpenGL functionality 
+        #define glGenVertexArrays glGenVertexArraysOES
```

Comment:
Why those functions pointers need to be defined here? rlgl should probably do that mapping internally.

Reply:
I will consider moving it to rlgl.h

Reply:
ok, nice